plugins/source/square/docs/_configuration.md

```diff
@@ -4,7 +4,7 @@ kind: source
 spec:
   name: square
   registry: docker
-  path: ghcr.io/cloudquery/square:VERSION_SOURCE_SQUARE
```

Comment:
There was a bug here, this should have been `cq-source-square`, see https://github.com/cloudquery/cloudquery/pkgs/container/cq-source-square